examples/ecs/parallel_query.rs

```diff
@@ -37,10 +37,13 @@ fn move_system(mut sprites: Query<(&mut Transform, &Velocity)>) {
 }
 
 // Bounce sprites outside the window
-fn bounce_system(windows: Res<Windows>, mut sprites: Query<(&Transform, &mut Velocity)>) {
-    let window = windows.primary();
-    let width = window.width();
-    let height = window.height();
+fn bounce_system(
+    resolutions: Query<&WindowResolution>,
```

Comment:
If we are using marker component for primary window then this should filter for the marker instead of "hoping" that there only is one window. 
```suggestion
    resolutions: Query<&WindowResolution, With<PrimaryWindow>>,
```

Reply:
Should be fixed for all these examples now to include that.